app/forms/new_article_form.rb

```diff
@@ -0,0 +1,12 @@
+class NewArticlesForm
```

Comment:
your class name doesn't match your filename!
file_name - app/forms/new_article_form.rb
class_name - NewArticlesForm -> see, you used plural here